Tiltfile

```diff
@@ -22,8 +22,8 @@ settings = {
     "kind_cluster_name": "capz",
     "capi_version": "v1.6.0",
     "cert_manager_version": "v1.13.2",
-    "kubernetes_version": "v1.28.3",
-    "aks_kubernetes_version": "v1.28.3",
+    "kubernetes_version": "v1.28.0",
```

Comment:
why this change?

Reply:
+1

(I assume this is cruft/rebase errata)

Reply:
Yup, leftover from testing. I was seeing if reverting the version would fix the test like it did for fleet (it did not)